modules/nf-core/seqtk/sample/main.nf

```diff
@@ -8,8 +8,7 @@ process SEQTK_SAMPLE {
         'quay.io/biocontainers/seqtk:1.3--h5bf99c6_3' }"
 
     input:
-    tuple val(meta), path(reads)
-    val sample_size
+    tuple val(meta), path(reads), val(sample_size)
```

Comment:
if the module is overhauled anyway, I would argue for removing this val and putting it in the task.ext.args... 

(I am scared already and will hide somewhere :D )

Reply:
The subworfklow is very new and only used (soon) in demultiplexing and my internal workflow - happy if this makes it more generic to use 👍🏻